.gitignore

```diff
@@ -112,6 +112,5 @@ ENV/
 tutorial/example.db
 docs/_build/
 docs/source/reference/generated/
-docs/source/reference/alias_generated/
```

Comment:
Maybe we can remove this directory from the target of `make clean`.

Reply:
Thank you! I fixed it.